docker-compose.yaml

```diff
@@ -23,6 +23,8 @@ services:
     environment:
       - DB_HOST=db
       - DATABASE_URL=postgresql://granola:systems@db:5432/db
+      - SERVER_LOG_LEVEL=info # This sets the highest level for logging allowed
```

Comment:
Does this actually work, or is it the RUST_LOG env var that does the trick?

Reply:
@robinbb if I set RUST_LOG in my .env file while using Docker nothing happens, but when I change the log level in these two code lines and then use Docker it changes. At least for Docker on my machine.

Reply:
@jstavh Something else is definitely happening, the var in this PR doesn't exist & so has no impact on log level.
RUST_LOG is the way to go. 👍